src/app/factories/utils.factory.js

```diff
@@ -0,0 +1,44 @@
+export function UtilsFactory($document) {
+  const module = {};
+
+  module.strFormat = function (str, formatObject) {
```

Comment:
Is there any reason you created this function, rather than just using ES6 template strings?

e.g.
```
let foo = 'a';
let bar = `${foo}bc`;
```